pkg/metrics/platform.go

```diff
@@ -117,8 +118,24 @@ func RegisterPlatformMetrics(_ context.Context, platform string) error {
 	return nil
 }
 
+// Convert the platform label to the format used by PlatformMetrics in case of a mismatch
+func platformLabel(platform string) string {
+	return strings.ReplaceAll(platform, "-", "/")
+}
+
+// Compare the platform label to the format used by PlatformMetrics and return the corresponding PlatformMetrics
+func platformLabelComperator(platformMetrics map[string]*PlatformMetrics, platform string) *PlatformMetrics {
+	if pmetrics, ok := platformMetrics[platform]; ok {
+		return pmetrics
+	}
+	if pmetrics, ok := platformMetrics[platformLabel(platform)]; ok {
+		return pmetrics
+	}
```

Comment:
I think this is a smoother way of doing it, while keeping the safety of fetching values from a map the way you did. WDYT?

```suggestion
	pmetrics, ok := platformMetrics[platform]

	if !ok {
		pmetrics, ok = platformMetrics[platformLabel(platform)]
	}
	if ok {
		return pmetrics
	}
```